sidebars.js

```diff
@@ -338,6 +338,11 @@ module.exports = {
                         label: "Use LoadBalancer Services",
                         id: "manage/kubernetes/networking/configure-external-access-loadbalancer",
                       },
+                      {
+                        type: "doc",
+                        label: "Use a Custom Service",
```

Comment:
Parallelism with `Use LoadBalancer Services`?

```suggestion
                        label: "Use Custom Services",
```